packages/core/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@actions/core",
-  "version": "1.4.0",
+  "version": "1.5.0",
```

Comment:
I believe we are already on `1.5.0`, so this might need to be updated in your main branch

Reply:
I can see 1.4.0 in main branch right now. If there is any PR with 1.5.0 version then should we wait for it to be merged before this PR? If yes, here version should be 1.5.1, right?

Reply:
Ha I had a PR open for it, but it hasn't gotten merged: https://github.com/actions/toolkit/pull/873